packages/fast-components/src/styles/patterns/button.ts

```diff
@@ -0,0 +1,199 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../display";
+import { focusVisible } from "../focus";
+import { heightNumber } from "../size";
+
+export const BaseButtonStyles = css`
+    ${display("inline-block")} :host {
+        font-family: var(--body-font);
+        outline: none;
+    }
+
+    .control {
+        ${
+            /* Font size is temporary - 
+            replace when adaptive typography is figured out */ ""
+        } font-size: 14px;
+        line-height: 20px;
```

Comment:
Probably remove the fixed line-height value and keep the "1".

Reply:
sounds good